package.json

```diff
@@ -87,17 +90,18 @@
     "typescript": "^4.1.3"
   },
   "scripts": {
-    "build": "yarn run compile-config-schema && tsc -p tsconfig.build.json",
+    "build:fat": "yarn run compile-config-schema && tsc -p tsconfig.build.json",
     "build:watch": "yarn run compile-config-schema && tsc -p tsconfig.build.json --watch",
+    "build": "yarn compile-config-schema && esbuild src/index.ts --bundle --platform=node --target=node10.8 --outfile=dist/craft --minify",
```

Comment:
This is the important part.

Reply:
👍 

```
$ yarn && yarn build
[...]
$ ./dist/craft 
craft <command>

Commands:
  craft prepare NEW-VERSION  🚢 Prepare a new release branch
                          [aliases: p, prerelease, prepublish, prepare, release]
  craft publish NEW-VERSION  🛫 Publish artifacts         [aliases: pp, publish]
  craft targets              List defined targets as JSON array
  craft artifacts <command>  📦 Manage artifacts          [aliases: a, artifact]

Options:
  --no-input     Suppresses all user prompts          [boolean] [default: false]
  --dry-run      Dry run mode: do not perform any real actions
                                                      [boolean] [default: false]
  -v, --version  Show version number                                   [boolean]
  -h, --help     Show help                                             [boolean]

Not enough non-option arguments: got 0, need at least 1
$
```

Reply:
Are all six of these warnings okay?

```
$ node ./scripts/config-json-schema-to-ts.js
 > node_modules/yargs/index.js:12:39: warning: Indirect calls to "require" will not be bundled (surround with a try/catch to silence this warning)
    12 │   const argv = yargs(processArgs, cwd, require)
       ╵                                        ~~~~~~~

 > node_modules/yargs/yargs.js:378:33: warning: Indirect calls to "require" will not be bundled (surround with a try/catch to silence this warning)
    378 │     const req = parentRequire || require
        ╵                                  ~~~~~~~

 > node_modules/update-notifier/index.js:5:42: warning: Indirect calls to "require" will not be bundled (surround with a try/catch to silence this warning)
    5 │ const importLazy = require('import-lazy')(require);
      ╵                                           ~~~~~~~

 > node_modules/require-main-filename/index.js:2:25: warning: Indirect calls to "require" will not be bundled (surround with a try/catch to silence this warning)
    2 │   _require = _require || require
      ╵                          ~~~~~~~

 > node_modules/yargs/lib/apply-extends.js:57:82: warning: This call to "require" will not be bundled because the argument is not a string literal (surround with a try/catch to silence this warning)
    57 │ ... isPath ? JSON.parse(fs.readFileSync(pathToDefault, 'utf8')) : require(config.extends)
       ╵                                                                   ~~~~~~~

 > node_modules/yargs-parser/build/index.cjs:1005:19: warning: This call to "require" will not be bundled because the argument is not a string literal (surround with a try/catch to silence this warning)
    1005 │             return require(path);
         ╵                    ~~~~~~~

6 warnings
```

Reply:
Also, I expected to be able to move the `craft` executable around and have it still work but it doesn't.

```
$ mv craft ~/Desktop
$ cd ~/Desktop
$ ./craft 
internal/modules/cjs/loader.js:883
  throw err;
  ^

Error: Cannot find module 'is-ci'
Require stack:
- /Users/chadwhitacre/Desktop/craft
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
    at Function.Module._load (internal/modules/cjs/loader.js:725:27)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at a5 (/Users/chadwhitacre/Desktop/craft:148:1046)
    at /Users/chadwhitacre/Desktop/craft:148:1129
    at new cce (/Users/chadwhitacre/Desktop/craft:148:2429)
    at ace.exports (/Users/chadwhitacre/Desktop/craft:152:494)
    at sze (/Users/chadwhitacre/Desktop/craft:610:2291)
    at Yus (/Users/chadwhitacre/Desktop/craft:649:699)
    at Object.<anonymous> (/Users/chadwhitacre/Desktop/craft:649:1229)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/Users/chadwhitacre/Desktop/craft' ]
}
$
```

What is the expected behavior here?

Reply:
>Are all six of these warnings okay?

Yes

>Also, I expected to be able to move the craft executable around and have it still work but it doesn't.

No, this means you are looking at an older version of this PR or something. (otherwise the docker image check would fail too with the same error)

Reply:
>No, this means you are looking at an older version of this PR or something. (otherwise the docker image check would fail too with the same error)

Oh, it _is_ failing. I'll fix this.